### DISCUSSION TITLE
archery 可以支持中英文切换吗?

### DISCUSSION THREAD

**Discussion by yangguocang** [hhyo/Archery#2810] [Ideas]
archery 可以支持中英文切换吗?   已上线archery作为数据库管理的核心平台,  但是我们是跨国公司 很多同事都是老外 不太能看懂中文,  不知是否可以在用户端配置中英文自动切换的功能呀

**Comment by LeoQuote** [comment]
重复 https://github.com/hhyo/Archery/discussions/2432
